subcommand/server-acl-init/policies_test.go

```diff
@@ -0,0 +1,289 @@
+package serveraclinit
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
```

Comment:
maybe make a note that you're only testing the policy methods that have templates so people (like me just now!) aren't wondering why some of the policy methods aren't being tested

Reply:
Switching the two unchanging methods to constants should make this more clear.